apps/server/src/controllers/userControllers/oauth.ts

```diff
@@ -0,0 +1,66 @@
+import { Request, Response } from 'express'
+import Env from '../../config/env'
+import { HTTP_STATUS_CODE } from '../../constants'
+import logger from '../../utils/logger'
+import { OAuth2Client } from 'google-auth-library'
+
+export const oAuthUrl = async (req: Request, res: Response) => {
+  try {
+    res.header('Access-Control-Allow-Origin', 'http://localhost:5173') // prevent blockage by CORS
```

Comment:
hardcoded value!

there's an ENV for that

I also believe this can be set in the cors({ ... }) in app.ts then removed here

Reply:
The env for this has port of 3000 which is different from this port. Should I create another env for it??